lib/preview-web/src/StoryIndexClient.ts

```diff
@@ -0,0 +1,18 @@
+import global from 'global';
+
+import { StoryIndex } from '@storybook/store';
+
+const { fetch, EventSource } = global;
+
+const PATH = './stories.json';
+
+export class StoryIndexClient extends EventSource {
```

Comment:
duplicate to above? not sure i understand why we need this anyway. is it for testing purposes?

Reply:
I need to figure out where a common spot is this should live. We need it for both the manager and the preview. 

Not sure what you mean by testing